Client/src/components/navbar.tsx

```diff
@@ -0,0 +1,186 @@
+import '../css/navbar.css';
+import { useState, useEffect } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useAuth0 } from '@auth0/auth0-react';
+import MediaQuery from 'react-responsive';
+import apiService from '../services/apiService';
+import { useAppDispatch, useAppSelector } from '../features/hooks';
+import { refreshData } from '../features/user_slice';
+
+function Navbar() {
+  const navigate = useNavigate();
+  const location = useLocation();
+  const [toggle, setToggle] = useState(false);
+  const [modalOpen, setModalOpen] = useState(false);
```

Comment:
What's the difference between toggle and modalOpen? I think they're always toggled at the same time so could be combined

Reply:
Done